python/bindings/sidekiq_tx_python.cc

```diff
@@ -37,6 +37,9 @@ void bind_sidekiq_tx(py::module& m)
         std::shared_ptr<sidekiq_tx>>(m, "sidekiq_tx", D(sidekiq_tx))
 
         .def(py::init(&sidekiq_tx::make),
+           py::arg("_card"),
+           py::arg("port_id"),
```

Comment:
should this be named `port_id1` since there is a `port_id2`?

Reply:
Same this is now fixed in the .h files, but it generated this code.